make-iso.sh

```diff
@@ -19,10 +19,15 @@ then
     echo "Please run this script as root (sudo $@$0)."
     exit
 fi
+# /var/lmc cannot exist before building
+if [ -d /var/lmc ]
+then
+    rm -rf /var/lmc
+fi
 dnf install -y lorax-lmc-novirt git vim-minimal pykickstart
 dnf install -y https://mirrors.ritlug.com/TigerOS/$vers/packages/x86_64/anaconda-installclass-tigeros-$vers-1.fc$vers.x86_64.rpm
 echo "${green}Beginning build process${reset}"
 setenforce 0
 livemedia-creator --ks kickstarts/tigeros.ks --no-virt --resultdir /var/lmc --project TigerOS-Live --make-iso --volid TigerOS --iso-only --iso-name TigerOS.iso --releasever $vers --title TigerOS-live --macboot
-echo "${green}ISO saved to $(pwd)/TigerOS.iso${reset}"
+echo "${green}ISO saved to /var/lmc/TigerOS.iso${reset}"
```

Comment:
update `/var/lmc` to a subset directory. Maybe have `/var/lmc/TigerOS-${date}` or something similar

Reply:
You didn't actually update it... you need to change the `--resultdir` arg

Reply:
@axk4545 I didn't make any changes yet :smile:

Reply:
I also just noticed my typo now after what? ... a year